ui/app/AppLayouts/Chat/views/ChatMessagesView.qml

```diff
@@ -18,17 +18,20 @@ import shared.status 1.0
 import shared.controls 1.0
 import shared.views.chat 1.0
 
+import AppLayouts.Profile.stores 1.0
+
 import "../controls"
 import "../panels"
+import "../stores"
```

Comment:
Needed? Given that you also added the typed import above

Reply:
Different store, "Profile" vs "Chat".  I'll import it as module, not directory.